### DISCUSSION TITLE
Auth: Custom vault file location

### DISCUSSION THREAD

**Discussion by JohnnyBrav0Sr** [ente-io/ente#2390] [Enhancements]
Hi

First and foremost I would like to express my appreciation for making and maintaining a great app! However, I would like to make the following feature request:

During the setup of the Auth app, I would like to pick any folder to store the encrypted Auth vault file, so that I can pick a folder that automatically synchronizes with my preferred cloud storage (using a cloud syncing app like MetaCtrl). That way, I wouldn't have to bother with exporting / importing backups, because on any device or user profile, the Auth vault would always be up to date automatically!

I use this setup for my Keepass vault file, and it works like a charm! Keepass DOES allow picking a custom vault file location during setup. I would really like to use the same method for my Auth vault, if only Auth would allow me to select a custom location for the Auth vault file during setup. (For security purposes, I would not use the same cloud storage used for my Keepass vault, even though both files are encrypted).

I would very much appreciate if you would consider this feature request. Thank you.

(With Keepass I mean KeepassXC for Windows and Keepass2Android for Android).

**Comment by bingoxo** [comment]
this is the only feature holding me from switching from Aegis to Ente auth

**Comment by ua741** [comment]
Duplicate https://github.com/ente-io/ente/discussions/1921
